clients/drcachesim/drpt2trace/drpt2trace.cpp

```diff
@@ -261,6 +264,29 @@ option_init(int argc, const char *argv[])
             return false;
         }
     }
+
```

Comment:
It would be helpful to add a comment here to describe the two modes: elf and sideband. Also mention in what cases each of them should be used (kernel and short user traces, vs others)

Reply:
Done.